enclave/sgx/collateralinfo.c

```diff
@@ -0,0 +1,186 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <openenclave/corelibc/stdio.h>
+#include <openenclave/enclave.h>
+#include <openenclave/internal/calls.h>
+#include <openenclave/internal/raise.h>
+#include <openenclave/internal/safecrt.h>
+#include <openenclave/internal/safemath.h>
+#include <stdlib.h>
+#include <string.h>
+#include "../common/sgx/collateral.h"
+#include "sgx_t.h"
+
+/**
+ * Call into host to fetch collateral information.
+ */
+oe_result_t oe_get_sgx_quote_verification_collateral(
```

Comment:
Same question here, how long does this ocall take?

Reply:
1,810,167 microseconds (around 1.8 seconds). By looking at the data, network communication with Azure PCK caching service took the majority of the time.

Reply:
I think you mean milliseconds.  We need to optimize this.  How long does the Azure DCAP client calls cost?